primitives/weights/src/weight_v2.rs

```diff
@@ -160,6 +160,26 @@ impl Weight {
 		Self { ref_time: 0 }
 	}
 
+	/// Constant version of Add with u64. Is only overflow safe when evaluated at compile-time.
```

Comment:
```suggestion
	/// Constant version of Add with u64.
	///
	/// Is only overflow safe when evaluated at compile-time.
```

By overflow safe at compile time you mean that it would panic?

Reply:
(I suggested the comment)

>By overflow safe at compile time you mean that it would panic?

Even better: it would not compile. For example `const _: Weight = Weight::MAX.add(u64::MAX);` does not compile.  
Whether or not it panics at runtime is dependent on the `overflow-checks` config in the `Cargo.toml`, or?

Reply:
At runtime it depends on `debug_assertions`